seismic/traveltime/test_ellipcorr.py

```diff
@@ -0,0 +1,26 @@
+"""
+Test if the ellipcorr lib has been installed correctly?
+
+1. Install ellip-corr into a python environment. See guide https://github.com/GeoscienceAustralia/ellip-corr/
+
+2. export  ELLIPCORR=~/Githubz/hiperseis/ellip-corr/
+
+3. run this script in the envionment where ellipcorr has been installed: 
+    source ~/Venvs/hiperseis/bin/activate
+
+"""
+
+import ellipcorr
+import numpy as np
+
+print(ellipcorr.ellipticity_corr('sP', 65, 124, 45, 39), "=? -0.38976147770881653")
```

Comment:
Consider putting all this into a test function. For development and debugging purposes, you can call test functions directly from inside a `if __name__ == "__main__":` block.